test/quantization/pt2e/test_quantize_pt2e.py

```diff
@@ -574,18 +576,14 @@ def annotate(self, model: torch.fx.GraphModule) -> torch.fx.GraphModule:
                         and node.target == torch.ops.aten.convolution.default
                     ):
                         input_act = node.args[0]
-                        assert isinstance(input_act, Node)
```

Comment:
Since this quantizer is only used for test case of `conv->maxpool`, I think removing it makes the code clean. Anyway, I add it back. Please help to take a look again.